### PR TITLE
CSS API Account Implementation and Email Notifications

### PR DESCRIPTION
**Pull request by NithinKuruba**
Fixes 
https://trello.com/c/wSN32LUF/1233-css-api-account-ui-implementation
https://trello.com/c/Ygrl1DK6/1438-css-api-account-email-notification

### CODE REVIEW

File: lambda/actions/src/controllers/batch.ts [hunk 1]
```diff
@@ -85,8 +87,21 @@ export const updatePlannedItems = async (data) => {
         (await models.event.count({ where: { eventCode: EVENTS.REQUEST_APPLY_SUCCESS, requestId: integration.id } })) >
         1;
 
-      const emailCode = isUpdate ? EMAILS.UPDATE_INTEGRATION_APPROVED : EMAILS.CREATE_INTEGRATION_APPROVED;
-      await sendTemplate(emailCode, { integration });
+      if (integration.apiServiceAccount) {
+        const teamIntegrations = await models.request.findAll({
+          where: { teamId: integration.teamId, apiServiceAccount: false, archived: false },
```
Comment (junminahn) [inline lambda/actions/src/controllers/batch.ts:92]: can we add the `attributes` option to select only necessary information?

File: lambda/shared/templates/helpers.ts [hunk 2]
```diff
@@ -94,3 +94,11 @@ export const getIntegrationEmails = async (integration: Data, primaryEmailOnly =
 
   return [];
 };
+
+export const processIntegrationList = async (integrations: Array<Data>) => {
```
Comment (junminahn) [inline lambda/shared/templates/helpers.ts:98]: wonder if the `one-liner` code can work as the same?
```
export const processIntegrationList =  (integrations: Array<Data>) => Promise.all(integrations.map(processRequest));
```

**Review by junminahn — COMMENTED** [review]
just minor comments; overall, looking good.

File: lambda/shared/templates/helpers.ts (hunk 2 above)
Comment (junminahn) [reply lambda/shared/templates/helpers.ts:98]: I believe we use `Type[]` as a convention for array type as well.

File: lambda/actions/src/controllers/batch.ts (hunk 1 above)
Comment (NithinKuruba) [reply lambda/actions/src/controllers/batch.ts:92]: done!

File: lambda/shared/templates/helpers.ts (hunk 2 above)
Comment (NithinKuruba) [reply lambda/shared/templates/helpers.ts]: sure updated them all to maintain the consistency!

**Review by junminahn — APPROVED** [review]

**Pull request merged**